### DISCUSSION TITLE
kpanda/best-practice/kubean-low-version/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#3564] [Announcements]
# kpanda/best-practice/kubean-low-version/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kpanda/best-practice/kubean-low-version/

**Comment by leolingg** [comment]
`制作离线增量包` 提示以下错误：
```shell
echo ${SPRAY_IMG_ADDR}
10.29.208.120:18082/daocloud-docker-v/cncp/ghcr.io/kubean-io/spray-job:2.21-e4ac3ab

podman run -it --rm -v $(pwd)/manifest.yml:/manifest.yml -v $(pwd)/data:/data -e ZONE=CN -e MODE=FULL ${SPRAY_IMG_ADDR} 
Error: no command or entrypoint provided, and no CMD or ENTRYPOINT from image
```
通过`podman inspect`命令可以看到进行中并没有CMD或者ENTRYPOINT指令。

**Reply by CoderTH** [reply]
已修复

**Comment by bill-ding007** [comment]
# 制作离线包，
SPRAY_IMG_ADDR="ghcr.m.daocloud.io/kubean-io/spray-job:2.21-d6f688f" 
这一步应该是 airgap-patch的镜像定义
